lstchain/scripts/lstchain_data_r0_to_dl1.py

```diff
@@ -12,7 +12,7 @@
 $> python lstchain_data_r0_to_dl1.py
 --input-file LST-1.1.Run02030.0000.fits.fz
 --output-dir ./
---pedestal-file drs4_pedestal.Run2028.0000.fits
+--pedestal-file drs4_pedestal.Run2028.0000.h5
```

Comment:
Why DRS4 pedestal file has now h5 format? I thought EVB team needed FITS files for the online correction.

Reply:
EVB needs a binary file that just contains the baseline values as uint16. They create it with a small python script from the FITS files and it will be as easy to create it from the h5 file.

Reply:
change introduced in #725